gui/main.js

```diff
@@ -11,9 +11,9 @@ const path = require('path')
 const autoLaunch = require('./src/main/autolaunch')
 const lastFiles = require('./src/main/lastfiles')
 const tray = require('./src/main/tray')
-const trayWindow = require('./src/main/tray.window.js')
-const helpWindow = require('./src/main/help.window.js')
-const onboardingWindow = require('./src/main/onboarding.window.js')
+const TrayWindowWM = require('./src/main/tray.window.js')
```

Comment:
Either `TrayWindow` or `TrayWindowManager`? I believe the first one would be ok.